### ISSUE TITLE
Error in the console.

### ISSUE THREAD

**Issue by K0-RR** [tr7zw/EntityCulling#38]
```
[13:21:25] [CullThread/INFO]: [STDERR]: java.lang.NullPointerException: Cannot invoke "dev.tr7zw.entityculling.access.Cullable.isForcedVisible()" because "cullable" is null
[13:21:25] [CullThread/INFO]: [STDERR]: 	at dev.tr7zw.entityculling.CullTask.run(CullTask.java:103)
[13:21:25] [CullThread/INFO]: [STDERR]: 	at java.base/java.lang.Thread.run(Thread.java:831)
[13:21:33] [CullThread/INFO]: [STDERR]: java.lang.NullPointerException: Cannot invoke "dev.tr7zw.entityculling.access.Cullable.isForcedVisible()" because "cullable" is null
[13:21:33] [CullThread/INFO]: [STDERR]: 	at dev.tr7zw.entityculling.CullTask.run(CullTask.java:103)
[13:21:33] [CullThread/INFO]: [STDERR]: 	at java.base/java.lang.Thread.run(Thread.java:831)
```

**Comment by tr7zw** [comment]
This isn't a critical thing to happen(as long as its not spammed) and has been fixed in 1.17 versions https://github.com/tr7zw/EntityCulling-Fabric/commit/5853c4e7c83ad46d0b5ff317f46bbfb6db996cd2 .

**Comment by K0-RR** [comment]
Sorry, somehow I missed the 1.3.3 release, could you add the CurseForge link to the readme?
